homeassistant/components/sensor/mold_indicator.py

```diff
@@ -0,0 +1,255 @@
+"""
+Calculates mold growth indication from temperature and humidity.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.mold_indicator/
+"""
+import logging
+import math
+
+import homeassistant.util as util
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import track_state_change
+from homeassistant.const import (ATTR_UNIT_OF_MEASUREMENT,
+                                 TEMP_CELCIUS, TEMP_FAHRENHEIT)
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = "Mold Indicator"
+CONF_INDOOR_TEMP = "indoor_temp_sensor"
+CONF_OUTDOOR_TEMP = "outdoor_temp_sensor"
+CONF_INDOOR_HUMIDITY = "indoor_humidity_sensor"
+CONF_CALIBRATION_FACTOR = "calibration_factor"
+
+MAGNUS_K2 = 17.62
+MAGNUS_K3 = 243.12
+
+ATTR_DEWPOINT = "Dewpoint"
+ATTR_CRITICAL_TEMP = "Est. Crit. Temp"
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    """Setup MoldIndicator sensor."""
+    name = config.get('name', DEFAULT_NAME)
```

Comment:
This configuration option is missing in your [PR](https://github.com/balloob/home-assistant.io/pull/311).